keras/utils/visualize_util.py

```diff
@@ -10,9 +11,11 @@
 except ImportError:
     # Fall back on pydot if necessary.
     import pydot
-if not pydot.find_graphviz():
-    raise ImportError('Failed to import pydot. You must install pydot'
-                      ' and graphviz for `pydotprint` to work.')
+try:
+    imp.find_module('graphviz')
+except:
```

Comment:
I just followed the original version and used ImportError. What error message do you think is proper.